### ISSUE TITLE
NTR: Liver biopsy: lysosomal-like PAS-positive vacuoles (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9162]
New term request Liver biopsy: lysosomal-like PAS-positive vacuoles (elthat_100720025009): Hepatic biopsy; periodic acid-Schiff

**Comment by pnrobinson** [comment]
@MickeySegal

is this Pompe?

Would the term actually efer to lymphocytes in the liver (and do we need a separate term for PAS-positive lymphocytes in different tissues?, i.e., do we also need https://github.com/obophenotype/human-phenotype-ontology/issues/9161)

How about
PAS-positive lymphocyte vacuolization
def: Presence of lymphocytes with positive staining for periodic acid-Schiff (PAS) cytoplasmic vaculoes
comment: Pompe disease (glycogen storage disease type II, acid maltase deficiency) is an inherited metabolic disorder in which a deficiency of the lysosomal enzyme acid α-glucosidase (EC 3.2.1.3/20) causes glycogen storage. Vacuolated lymphocytes - in essence, lysosomes filled with nondegraded material occur in many storage disorders. Glycogen storage in Pompe disease is found in lysosomes throughout the body, including lymphocytes in peripheral blood, liver, and skin.

**Comment by MickeySegal** [comment]
We got this initially from https://www.ncbi.nlm.nih.gov/books/NBK11168/ "Histologic examination of skin, liver, trachea, and heart shows lysosomal-like PAS-positive vacuoles, suggestive of glycoprotein and a storage disorder."  We also use it for Pompe: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2861182/ "PAS-positive lymphocyte vacuoles can be used as diagnostic screening test for Pompe disease".

So keeping the term general to multiple tissues seems like the best approach, as you say "found in lysosomes throughout the body, including lymphocytes in peripheral blood, liver, and skin".

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000639
- label: PAS-positive lymphocyte vacuolization
- parents: [Vacuolated lymphocytes (HP:0001922)](https://hpo.jax.org/app/browse/term/HP:0001922)
- pmid: PMID:20107902
- def: Presence of lymphocytes with positive staining for periodic acid-Schiff (PAS) cytoplasmic vacuoles.
- comment: Pompe disease (glycogen storage disease type II, acid maltase deficiency) is an inherited metabolic disorder in which a deficiency of the lysosomal enzyme acid alpha-glucosidase (EC 3.2.1.3/20) causes glycogen storage. Vacuolated lymphocytes - in essence, lysosomes filled with nondegraded material occur in many storage disorders. Glycogen storage in Pompe disease is found in lysosomes throughout the body, including lymphocytes in peripheral blood, liver, and skin.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9162
- synonyms:
